tests/fuser_basic_test.cc

```diff
@@ -72,152 +72,152 @@ TEST(FuserBasicTest, NoTimesToSplitAt) {
 
   EXPECT_EQ(fused_gates[0].kind, kGateCZ);
   EXPECT_EQ(fused_gates[0].time, 1);
-  EXPECT_EQ(fused_gates[0].num_qubits, 2);
+  EXPECT_EQ(fused_gates[0].qubits.size(), 2);
```

Comment:
Suggestion: with the amount of repetition in this file, it seems like it would be worthwhile to define equality operators for gates and fused gates. Doesn't need to be a part of this PR, though.

Reply:
Acknowledged.